plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -99,6 +101,23 @@ protected DistributedQueryRunner createQueryRunner()
         }
     }
 
+    @Test
+    public void testCreateTableAsSelect()
```

Comment:
Could you add a test for a CTAS on a partitioned table ?
I'm guessing that this test would be a bit more revealing.